scripts/upgrade

```diff
@@ -34,10 +34,10 @@ fpm_usage=$(ynh_app_setting_get --app=$app --key=fpm_usage)
 
 upgrade_type=$(ynh_check_app_version_changed)
 
-# Check machine architecture (Seens 25.0 Nextcloud doesn't support 32bit machines)
+# Check machine architecture (Nextcloud will soon deprecate 32-bit support)
 if [ $YNH_ARCH == "i386" ] || [ $YNH_ARCH == "armhf" ]
 then
-    ynh_die --message="Sorry, Nextcloud has deprecated 32-bit support"
+    ynh_print_warn --message="Nextcloud will soon deprecate 32-bit support. It is recommended to upgrade to a 64-bit architecture."
 fi
```

Comment:
Nextcloud will deprecate 32-bit support within the next years. You should start thinking about upgrading to a 64-bit architecture.